src/components/controls/controls.js

```diff
@@ -0,0 +1,222 @@
+var registerControls = require('../../core/controls').registerControls;
+var THREE = require('../../lib/three');
+
+var MAX_DELTA = 0.2; // ms
+var PI_2 = Math.PI / 2;
+
+/**
+ * Controls component.
+ *
+ * Receives input events from device-specific control components, and applies movement and rotation
+ * to the element accordingly.
+ */
+module.exports.Component = registerControls('controls', {
+
+  dependencies: ['velocity', 'rotation'],
+
+  schema: {
+    enabled: { default: true },
+
+    flyingEnabled: { default: false },
+
+    movement: { default: ['gamepad-controls', 'wasd-controls', 'touch-controls'] },
+    movementEnabled: { default: true },
```

Comment:
I see. I was thinking you would just exclude those controls from `movement`, but this allows syntax that is nicer and more consistent.